helsingborg-stad/helsingborg-io-sls-api
# Feat/upgrade serverless to v3

Upgrade Serverless from 2 to 3.2

## How to test

Concrete example:

1. Checkout this branch
2. run `sls deploy` within every service folder for `service x`
3. Verify no errors when deploying to your sandbox
4. Use Postman or eq. and make requests to selected endpoints
